src/Microsoft.PowerShell.Commands.Management/commands/management/ComputerUnix.cs

```diff
@@ -105,7 +105,7 @@ public class CommandLineCmdletBase : PSCmdlet, IDisposable
         /// </summary>
```

Comment:
@vexx32, your last commit had 1 failures in `PowerShell-CI-windows`
Job Cmdlet Tests.Retrieving partial output from jobs.Receive-Job will retrieve partial output, including -Keep results
```powershell
Destination array was not long enough. Check the destination index, length, and the array's lower bounds. (Parameter 'destinationArray')
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Core\Job.Tests.ps1: line 108
108:                         $results = Receive-Job -Keep $job

```